mappings/net/minecraft/inventory/DoubleInventory.mapping

```diff
@@ -1,4 +1,8 @@
 CLASS agv net/minecraft/inventory/DoubleInventory
 	FIELD a first Lagw;
 	FIELD b second Lagw;
-	METHOD a isPart (Lagw;)Z
+	METHOD <init> (Lagw;Lagw;)V
+		ARG 1 first
+		ARG 2 second
+	METHOD a contains (Lagw;)Z
```

Comment:
`contains` is wrong, it should stay `isPart`. An inventory can't contain another inventory since an inventory isn't an item. Someone could also think that "contains" means "contains all the items that the other inventory contains".

Reply:
How about `includes`?

Reply:
"includes X" makes me think even more of "contains everything that X contains". The two chest blocks (inventories) that make up a chest (double inventory) are called "parts". What's wrong with `isPart`?